persistence/sql/migrations/sql/20241001000000000000_identity_verifiable_addresses_pk.mysql.down.sql

```diff
@@ -0,0 +1,7 @@
+ALTER TABLE identity_verifiable_addresses
+    DROP FOREIGN KEY identity_verifiable_addresses_ibfk_1;
```

Comment:
I guess we need this because the primary key won't be deleted if there is still a foreign key using it, correct?

Reply:
exactly.